store.html

```diff
@@ -13,10 +13,13 @@
 
 <body>
     <header id="header_store">
+        <div>
             <svg id="logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 144 216">
                 <path fill="#F15B2A"
                     d="M126.1 212.9c4.1 4.1 10.7 4.1 14.8 0 4.1-4.1 4.1-10.7 0-14.8l-28.6-28.6 18.1-18.1c8.4-8.4 13.6-20 13.6-32.9C144 92.8 123.2 72 97.5 72S51 92.8 51 118.5c0 12.8 5.2 24.5 13.6 32.9l18.1 18.1-18.1 18.1c-4.6 4.6-11 7.5-18.1 7.5-14.1 0-25.6-11.5-25.6-25.6v-159C20.9 4.7 16.2 0 10.5 0 4.7 0 0 4.7 0 10.5v159C0 195.2 20.8 216 46.5 216c12.8 0 24.5-5.2 32.9-13.6l18.1-18.1 28.6 28.6zm-46.7-76.3c-4.6-4.6-7.5-11-7.5-18.1 0-14.1 11.5-25.6 25.6-25.6 14.1 0 25.6 11.5 25.6 25.6 0 7.1-2.9 13.5-7.5 18.1l-18.1 18.1-18.1-18.1z" />
```

Comment:
What is it happening here line 17-19?
Can this break your code?

Reply:
I have no idea :) , this was a script that existed before I cloned it. I used the logo offered by CYF without changing it. it should be the same in your repository.